samples/Mvc.DynamicCorsPolicyProvider/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.FileProviders.Physical": "1.2.0-*",
```

Comment:
I didn't know about this meta package but recall some earlier conversations now. Much better now. Thanks.